deps/rabbit_common/test/rabbit_env_SUITE.erl

```diff
@@ -1080,7 +1080,8 @@ check_parse_conf_env_file_output(_) ->
          "DOUBLE_QUOTED" => "c",
          "SINGLE_DOLLAR" => "d"},
        rabbit_env:parse_conf_env_file_output2(
-         ["UNQUOTED=a",
+         ["++ ؎ ", %% note unicode char
```

Comment:
You should have a test that produces a value you can assert against as well.

Reply:
Well I added the test, but because it's erlang string - kinda ugly (the array).

Reply:
This is a [Unicode character](https://unicode-table.com/en/060E/). The whole issue is likely about non-ASCII characters.

We can use something a bit more readable than that, too: a short Mandarin, Cyrillic or Hindi word.